### PR TITLE
[alertmanager] - Upgrade pdb version to apiVersion: policy/v1 

### LINKED ISSUE #1967: [prometheus] PodDisruptionBudget policy/v1beta1 deprecated from k8s 1.22+
### Is your feature request related to a problem ?

from https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125:
_The policy/v1beta1 API version of PodDisruptionBudget will no longer be served in v1.25.
Migrate manifests and API clients to use the policy/v1 API version, available since v1.21._

`prometheus/charts/kube-state-metrics/templates/pdb.yaml-apiVersion: policy/v1beta1`
`prometheus/templates/alertmanager/pdb.yaml-apiVersion: policy/v1beta1`
`prometheus/templates/pushgateway/pdb.yaml-apiVersion: policy/v1beta1`
`prometheus/templates/server/pdb.yaml-apiVersion: policy/v1beta1`

### Describe the solution you'd like.

also from https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125:
_All existing persisted objects are accessible via the new API_

therefore, something like this

```
{{ if $.Capabilities.APIVersions.Has "policy/v1/PodDisruptionBudget" -}}
apiVersion: policy/v1
{{- else -}}
apiVersion: policy/v1beta1
{{- end }}
kind: PodDisruptionBudget
```
should work.

### Describe alternatives you've considered.

NONE

### Additional context.

_No response_

### PR DESCRIPTION
**Pull request by haijun2022**
Signed-off-by: Zhang, Henry <Haijun.Zhang@fmr.com>

#### What this PR does / why we need it:
The policy/v1beta1 API version of PodDisruptionBudget will no longer be served in v1.25.
Migrate manifests and API clients to use the policy/v1 API version, available since v1.21.

#### Which issue this PR fixes
  - fixes #1967 

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [x] Chart Version bumped
- [x] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

**Review by monotek — CHANGES_REQUESTED** [review]
chart version has to be raised...

**Comment by haijun2022** [comment]
> chart version has to be raised...

Thanks for the comments, fixed!

File: charts/alertmanager/Chart.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ icon: https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/a
 sources:
   - https://github.com/prometheus/alertmanager
 type: application
-version: 0.18.0
+version: 0.19.0
```
Comment (monotek) [inline charts/alertmanager/Chart.yaml:9]:
```suggestion
version: 0.18.1
```

**Review by monotek — CHANGES_REQUESTED** [review]

**Review by monotek — CHANGES_REQUESTED** [review]
Soory, it had to be 0.18.2

**Comment by haijun2022** [comment]
> 

no worries, I am going to update it now, thank you for your quick review.

**Review by monotek — CHANGES_REQUESTED** [review]
can you please rebase too / fix the linting errors

**Pull request closed without merging**